src/js/helpers/fetchAvatar.js

```diff
@@ -30,12 +30,12 @@ module.exports = function (jid, id, type, source, cb) {
                         return cb(fallback(jid));
                     }
 
-		            if (!resp.vCardTemp.photo) return cb(fallback(jid));
+                    if (!resp.vCardTemp.photo) return cb(fallback(jid));
 
                     type = resp.vCardTemp.photo.type || type;
 
-                    var data = resp.vCardTemp.photo.data;
-                    var uri = 'data:' + type + ';base64,' + data;
+                    var data = resp.vCardTemp.photo.data,
+                        uri = 'data:' + type + ';base64,' + data;
```

Comment:
Here (and everywhere else) I honestly prefer the style when every variable has its own qualifier (`var`, `let`, `const` etc.).